docs/commands/alias.md

```diff
@@ -0,0 +1,62 @@
+# alias
+This command allows you to define shortcuts for common commands, equivalent to the same bash command.
+
+The command expects three parameters:
+* the name of alias
+* the parameters as a space-separated list (`[a b ...]`), can be empty (`[]`)
```

Comment:
"space-separated" could probably be dropped, since that's just how we express a list/table in nushell. If we haven't described that elsewhere, we should 🙂

Reply:
I added it after trying `[a, b]` first. I think in this case a little bit of redundancy is good.

If you think it really doesn't belong here, I suggest adding an example with multiple parameters instead. Though I'd need some idea for that, since all my bash alias take no arguments or forward them to the underlying command instead (`d --staged` => `git diff --staged`).

Reply:
It should be good for now. I'm hoping documentation/books gets better, so that by the time one gets here, one also has the fundamentals of nushell syntax 🙂